docs/server/source/production-deployment-template/add-node-on-kubernetes.rst

```diff
@@ -74,10 +86,54 @@ Claims, and to run MongoDB in the new cluster:
 1. :ref:`Add Storage Classes <Step 10: Create Kubernetes Storage Classes for MongoDB>`.
 2. :ref:`Add Persistent Volume Claims <Step 11: Create Kubernetes Persistent Volume Claims>`.
 3. :ref:`Create the Config Map <Step 3: Configure Your BigchainDB Node>`.
-4. :ref:`Run MongoDB instance <Step 12: Start a Kubernetes StatefulSet for MongoDB>`.
+4. :ref:`Prepare the Kubernetes Secrets <Step 3: Configure Your BigchainDB Node>`, as per your
+   requirement i.e. if you do not want a certain functionality, just remove it from the
+   ``configuration/secret.yaml``.
+5. :ref:`Run MongoDB instance <Step 12: Start a Kubernetes StatefulSet for MongoDB>`.
+
+
+Step 3: Start NGINX service, Assign DNS to NGINX Pubic IP and run NGINX deployment
+----------------------------------------------------------------------------------
+
+Please see the following pages:
+
+* :ref:`Start NGINX service <Step 4: Start the NGINX Service>`.
+* :ref:`Assign DNS to NGINX Public IP <Step 5: Assign DNS Name to the NGINX Public IP>`.
+* :ref:`Run NGINX deployment <Step 9: Start the NGINX Kubernetes Deployment>`.
+
+
+Step 4: Verify network connectivity between the MongoDB instances
+-----------------------------------------------------------------
 
+Make sure your MongoDB instances can access each other over the network. *If* you are deploying
+the new MongoDB node in a different cluster or geographical location using Azure Kubernetes container
+service, you will have to set up networking (name resolution) between the two clusters using `Kubernetes
```

Comment:
Also, "set up networking (name resolution)" is not necessary, as there might be cases where a public IP might also be used.